README.md

```diff
@@ -150,6 +150,43 @@ cosign verify \
 
 If the signature verification did not result in an error, the deployment of Sigstore was successful!
 
```

Comment:
in a follow-up, we might move the top section to a new markdown file, and keep this new content as the default that everyone will see when they land here.